src/pybind/mgr/rgw/module.py

```diff
@@ -207,9 +208,26 @@ def _cmd_rgw_realm_bootstrap(self,
         try:
             for spec in rgw_specs:
                 self.create_pools(spec)
-                RGWAM(self.env).realm_bootstrap(spec, start_radosgw)
+                RGWAM(self.env).realm_bootstrap(spec, start_radosgw, resume)
+
         except RGWAMException as e:
             self.log.error('cmd run exception: (%d) %s' % (e.retcode, e.message))
+            if rgw_specs:
+                spec = rgw_specs[0]
+                existing = []
+                if spec.rgw_realm in RGWAM(self.env).realm_op().list():
+                    existing.append(f"realm: {spec.rgw_realm}")
+                if spec.rgw_zonegroup in RGWAM(self.env).zonegroup_op().list():
+                    existing.append(f"realm: {spec.rgw_zonegroup}")
+                if spec.rgw_zone in RGWAM(self.env).zone_op().list():
+                    existing.append(f"realm: {spec.rgw_zone}")
+
+                if existing:
+                    e.stderr = (e.stderr or '') + (
+                        f"\nNote: Partial bootstrap detected – {', '.join(existing)} already exist.\n"
+                        f"To resume, run:\n  ceph rgw realm bootstrap --resume -i <spec.yaml>\n"
```

Comment:
```suggestion
                        f"To resume, run:\n  ceph rgw realm bootstrap with --resume\n"
```
(or with whatever you change the flag name to, given my other comment). Just since you can technically run this command by just specifying `--realm-name <realm-name> --zone-name <zone-name>` etc. we should just say to add the flag rather than explicitly including to use `-i <spec.yaml>` in the suggestion